### ISSUE TITLE
Sonarr throws error when trying to get metadata images for a Series added through Jellyseer

### ISSUE THREAD

**Issue by ianabney** [Fallenbagel/jellyseerr#337]
### Description

When I add a series to sonarr using a Jellyseer request, the show is added to sonarr but when sonarr tries to get the metadata images (poster, banner, fanart) it throws an error and doesn't download the images. When I add the series through sonarr directly this problem does not happen, also when I delete a series that was added through jellyseer and then add it back through sonarr, the problem is fixed.
Here is a pastebin of the sonarr trace log for when it fails to download metadata images: https://pastebin.com/TAxvGncb

I can provide a docker compose or settings used if needed.

### Version

1.4.1

### Steps to Reproduce

1. Add a new series using the request on Jellyseer
2. Wait for sonarr to add the series and download the show
3. Sonarr adds media files fine, but doesn't grab metadata images.
4. Delete series from Sonarr (not files)
5. Re-add series using sonarr
6. Sonarr finds and grabs metadata images.

### Platform

desktop

### Device

Desktop Server

### Operating System

Ubuntu

### Browser

Firefox

### Code of Conduct

- [X] I agree to follow Overseerr's Code of Conduct

**Comment by ianabney** [comment]
So I did some more testing and I found out this only happens when I add a normal (non-anime) show for whatever reason. When I add an anime show that uses different tags, root folder, and quality profile; Sonarr can find and add the metadata images. I've attached the Jellyseer settings for sonarr showing the different tags/root folders/quality. When I hit test it passes so I don't think the tags or root folder is wrong for normal shows.

![image](https://user-images.githubusercontent.com/39421160/221454516-1ccc5f47-d235-4203-851d-1e99d5c74d9e.png)
